File: PlayerUI/Flutter/objc/FlutterPlayer/BCOVFlutterPlugin.m
//
//  BCOVFlutterPlugin.m
//  FlutterPlayer
//
//  Copyright © 2024 Brightcove, Inc. All rights reserved.
//

#import "BCOVFlutterPlugin.h"

#import "BCOVVideoPlayerFactory.h"


@implementation BCOVFlutterPlugin

#pragma mark - FlutterPlugin

+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar
{
    [registrar registerViewFactory:[BCOVVideoPlayerFactory new]
                            withId:@"bcov.flutter/player_view"];
}

@end

File: PlayerUI/Flutter/FlutterPlayer/BCOVFlutterPlugin.swift
//
//  BCOVFlutterPlugin.swift
//  FlutterPlayer
//
//  Copyright © 2026 Brightcove, Inc. All rights reserved.
//

import UIKit
import Flutter


final class BCOVFlutterPlugin: NSObject, FlutterPlugin {

    static func register(with registrar: FlutterPluginRegistrar) {
        registrar.register(BCOVVideoPlayerFactory(), withId: "bcov.flutter/player_view")
    }
}
